crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -204,6 +204,9 @@ pub struct StandardMaterial {
     /// shadows, alpha mode and ambient light are ignored if this is set to `true`.
     pub unlit: bool,
 
+    /// Whether to disable fog for this material.
+    pub no_fog: bool,
```

Comment:
We can, I used a negative form mainly to make it consistent with `unlit`, and also because I thought maybe novice users would be confused with setting `fog_enabled: true` and not getting any fog.

BTW, do you think a per-entity setting would make sense, like `NotShadowCaster`, i.e. `NotFogAffected` instead of, _or in addition to_, a material flag?

Reply:
I think setting no_fog: false is more confusing. :)

A per-mesh entity flag could be good but it could be done in a follow up PR if you prefer.

Reply:
> `fog_enabled: true` and not getting any fog.

Couldn't you just invert it were you are using it?

Reply:
Changed it, it's now a positive rather than a negative flag. Yeah, I think I prefer to do the entity flag on a follow up PR, this branch/thread is already running quite long, it's getting somewhat hard to follow.